jedeschule/pipelines/school_pipeline.py

```diff
@@ -37,21 +37,24 @@ def process_item(self, item, spider):
                             website=item.get('website'),
                             fax=item.get('fax'),
                             email=email,
-                            address=address)
+                            address=address
+                            )
```

Comment:
Those bracket changes are not required for the PR, but im kind of okay with it.

Reply:
In order to keep this PR small, I'd actually prefer those brackets not to change.